### PR TITLE
Add recent extensions platform changes

### PR DESCRIPTION
**Pull request by dotproto**
This PR adds two recent updates to the extensions what's new page

* Chrome 100: Native messaging port keeps service worker alive
* Chrome 100: omnibox.setDefaultSuggestion() supports promises and callbacks

[Live server](https://developer.chrome.com/docs/extensions/whatsnew)
[Preview server](https://deploy-preview-2120--developer-chrome-com.netlify.app/docs/extensions/whatsnew)

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ updated: 2022-01-20
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 100: Native messaging port keeps service worker alive {: #m100-native-msg-lifetime }
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:19]:
```suggestion
### Chrome 100: native messaging port keeps service worker alive {: #m100-native-msg-lifetime }
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ updated: 2022-01-20
...
+
+February 9, 2022
+
+Connecting to a native messaging host using chrome.runtime.connectNative() in an extension's service
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:23]:
```suggestion
Connecting to a native messaging host using `chrome.runtime.connectNative()` in an extension's service
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 3]
```diff
@@ -16,6 +16,22 @@ updated: 2022-01-20
...
+
+### Chrome 100: omnibox.setDefaultSuggestion() supports promises and callbacks {: #m100-omnibox-setdefault }
+
+February 8, 2022
+
+The
+[`omnibox.setDefaultSuggestion()`](/docs/extensions/reference/omnibox/#method-setDefaultSuggestion)
+method now returns a promise or accepts a callback to allow developers to determine when the
+suggestion has been properly set.
+
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:34]: Please remove the back ticks from the heading below. The explanation is in the other PR I reviewed this morning.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md (hunk 3 above)
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:34]: Roger, I'll update this in both PRs.

File: site/en/docs/extensions/whatsnew/index.md [hunk 4]
```diff
@@ -16,6 +16,22 @@ updated: 2022-01-20
...
 ### Chrome 100: `i18n.getMessage()` support in extension service workers {: #m100-i18n-getmessage }
```
Comment (dotproto) [inline site/en/docs/extensions/whatsnew/index.md:35]:
```suggestion
### Chrome 100: i18n.getMessage() support in extension service workers {: #m100-i18n-getmessage }
```

**Comment by dotproto** [comment]
Feedback from last review has been addressed.

**Pull request merged**